### PR TITLE
Moving terminology into one section

### LINKED ISSUE #42: Move terminology into one section
ekr had the following comment:

    S 2.4.
    Why do you have two terminology sections? I would merge them.

### PR DESCRIPTION
**Pull request by seanturner**
Closes #42.

### CODE REVIEW

**Review by fluffy — APPROVED** [review]

File: rtcweb-overview.xml [hunk 1]
```diff
@@ -163,33 +163,95 @@
...
 
-        <t><list style="symbols">
-            <t>A WebRTC browser (also called a WebRTC User Agent or WebRTC UA)
-            is something that conforms to both the protocol specification and
-            the Javascript API cited above.</t>
+        <t><list style="hanging">
+            <t hangText="Agent:">Undefined term. See "SDP Agent" and "ICE
+            Agent".</t>
+             
+            <t hangText="API:">Application Programming Interface - a
```
Comment (martinthomson) [inline rtcweb-overview.xml]: Be consistent about using "-" vs "."

File: rtcweb-overview.xml [hunk 2]
```diff
@@ -163,33 +163,95 @@
...
+            target="W3C.WD-html5-20110525"/>. See also "WebRTC User
+            Agent".</t>
+              
+            <t hangText="Data channel:">An abstraction that allows data to be
+            sent between WebRTC endpoints in the form of messages. Two
+            endpoints can have multiple data channels between them.</t>
+              
+            <t hangText="ICE Agent:">An implementation of the Interactive
+            Connectivity Establishment (ICE) <xref
+            target="I-D.ietf-ice-rfc5245bis"/> protocol. An ICE Agent may also
```
Comment (martinthomson) [inline rtcweb-overview.xml:191]: Ok, so which one is it?  5245, or 5245bis?

File: rtcweb-overview.xml [hunk 3]
```diff
@@ -163,33 +163,95 @@
...
+            specification of a set of calls and events, usually tied to a
+            programming language or an abstract formal specification such as
+            WebIDL, with its defined semantics.</t>
+              
+            <t hangText="Browser:">Used synonymously with "Interactive User
+            Agent" as defined in the HTML specification <xref
+            target="W3C.WD-html5-20110525"/>. See also "WebRTC User
+            Agent".</t>
+              
+            <t hangText="Data channel:">An abstraction that allows data to be
```
Comment (martinthomson) [inline rtcweb-overview.xml]: You seem to use "Title Case" elsewhere.

**Review by martinthomson — APPROVED** [review]
Most of my comments are consequential.

File: rtcweb-overview.xml (hunk 2 above)
Comment (seanturner) [reply rtcweb-overview.xml:191]: Definitely 5245, but this change is being incorporated via #37.

**Pull request merged**